bundles/org.openhab.core/src/main/java/org/openhab/core/internal/items/ExpireManager.java

```diff
@@ -166,7 +170,7 @@ private void processEvent(String itemName, Type type) {
     private @Nullable ExpireConfig getExpireConfig(String itemName) {
         Optional<ExpireConfig> itemConfig = itemExpireConfig.get(itemName);
         if (itemConfig != null) {
-            return itemConfig.isPresent() ? itemConfig.get() : null;
+            return itemConfig.orElse(null);
```

Comment:
return itemConfig.get() may be enough. You should remove the Nullable annotation on the itemExpireConfig map value. As it is a ConcurrentHashMap, null values are not allowed.

Reply:
Mistake on my side, you do need the code you put in, but you can still remove the @Nullable annotation from the declaration.